package.json

```diff
@@ -68,7 +68,8 @@
     "webpack-assets-manifest": "5",
     "webpack-cli": "4",
     "webpack-dev-middleware": "^5.3.4",
-    "webpack-merge": "5"
+    "webpack-merge": "5",
+    "ws": "8.17.1"
```

Comment:
Why are we adding a library as part of a vulnerability fix?

Reply:
Correct. Trying to redo the PR because having some issue with merge.